### ISSUE TITLE
Search does not return results - Search.loadIndex is not a function

### ISSUE THREAD

**Issue by fhennecker** [rtfd/sphinx_rtd_theme#743]
### Problem

I'm experiencing an issue where the search results page hangs (just like #348 )
I have tried several options (including installing manually by git cloning the theme).

#### Error Logs/Results
I have an error in the Chrome console:

```
Uncaught TypeError: Search.loadIndex is not a function
    at HTMLDocument.<anonymous>
```

### Environment Info

Using python `3.7.1`, sphinx `2.0.0`, sphinx-rtd-theme==`0.4.3 `

**Comment by Blendify** [comment]
Did this work before sphinx 2.0.0?

**Comment by hanzwo** [comment]
See here [https://github.com/sphinx-doc/sphinx/issues/6244](url).

**Comment by Blendify** [comment]
Adding proper link: https://github.com/sphinx-doc/sphinx/issues/6244

This seems to be a bug in Sphinx but will keep this report open to make sure everything works when sphinx fixes itself.

**Comment by Blendify** [comment]
https://github.com/rtfd/sphinx_rtd_theme/pull/729/ Should also fix this problem

**Comment by fhennecker** [comment]
Sphinx fixed this in their 2.0.1 version, it's working now!
